operators/README.md

```diff
@@ -1,53 +1,100 @@
-# CrownLabs operators
```

Comment:
Would it be possible to merge a bit the two README files? Just to make it a bit more general (as it is no longer the lab-operator README file), as well as to make sure all instructions still work (many commands appear out of date).